scripts/launch.sh

```diff
@@ -1,6 +1,13 @@
 #!/bin/sh
 
+export HOSTNAME_IP_ADDRESS=$(hostname -i)
+
 CONFIG_FILE=${QDROUTERD_HOME}/etc/qdrouterd.conf
+CONFIG_MAP_FILE=/etc/qpid-dispatch/qdrouterd.conf.template
+
+if [ -f $CONFIG_MAP_FILE ]; then
+    DOLLAR='$' envsubst < $CONFIG_MAP_FILE > $CONFIG_FILE
```

Comment:
Ideally this would replace https://github.com/interconnectedcloud/qdrouterd-image/blob/master/scripts/configure.sh#L38